src-chrome-mv3/options.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Popup</title>
+    <style>
+      dt { white-space: nowrap }
+    </style>
+    <script src="options.js"></script>
+  </head>
+  <body>
+    <h1>Imageshare Search</h1>
+    <input type="search" id="search" name="imgs"
+    aria-label="Search for resource matches in Imageshare">
```

Comment:
An `aria-label` doesn't need to be added if a `label` for the `input` already exists. If "Search" is too limited a term, I'd suggest "Search resources", but adding context should not be achieved via special AT labels. Either use a heading or other leading contextual text for that ("Search the Imageshare accessible STEM resource database"). A good rule is to see what context could be added in general when you find yourself resorting to "AT-specific" additions.

Reply:
change enacted in alerts-cross-browser